src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.cs

```diff
@@ -95,5 +108,71 @@ internal abstract partial class AbstractFindUsagesService : IFindUsagesService
 
             return progressAdapter;
         }
+
+        private async Task<bool> TryFindLiteralReferencesAsync(
+            Document document, int position, IFindUsagesContext context)
+        {
+            var cancellationToken = context.CancellationToken;
+            cancellationToken.ThrowIfCancellationRequested();
+
+            var syntaxTree = await document.GetSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
+            var syntaxFacts = document.GetLanguageService<ISyntaxFactsService>();
+            var token = await syntaxTree.GetTouchingTokenAsync(
+                position,
+                t => syntaxFacts.IsNumericLiteral(t) ||
+                     syntaxFacts.IsCharacterLiteral(t) ||
+                     syntaxFacts.IsStringLiteral(t),
+                cancellationToken).ConfigureAwait(false);
+
+            if (token.RawKind == 0)
+            {
+                return false;
+            }
+
+            // Searching for decimals not supported currently.
+            var tokenValue = token.Value;
+            if (tokenValue == null || tokenValue is decimal)
```

Comment:
this is because, right now, we encode numeric literals as 'longs' and index those.  a decimal might be longer than a long, so we can't store them.  We could store an additional index for these, but i didn't thnk it was important enough.